packages/app-project/stores/User/User.spec.js

```diff
@@ -37,19 +32,31 @@ describe('stores > User', function () {
     .get('/project_preferences?project_id=1&user_id=1&http_cache=true')
     .reply(200, {
       project_preferences: [
-        { activity_count: 23 }
+        {
+          links: {
+            user: '1'
+          }
```

Comment:
`activity_count` is not used for anything in FEM anymore, so don't mock it anywhere.